CONTRIBUTING.md

```diff
@@ -40,6 +42,41 @@ make install-requirements
 pip install -e '.[complete]'
 ```
 
+## Developing with ARM64 machines
+
+Some mlrun dependencies are not yet available for ARM64 machines via pypi, so we need to work with conda to get the packages compiled for ARM64 platform.   
+Install Anaconda from [here](https://docs.anaconda.com/free/anaconda/install/index.html) and then follow the steps below:
+
+Fork, clone and cd into the MLRun repository directory
+```shell script
+git clone git@github.com:<your username>/mlrun.git
+cd mlrun
+```
+
+Create a conda environment and activate it
+```shell script
+conda create -n mlrun python=3.9
+conda activate mlrun
+``` 
+
+Then, install the dependencies
+```shell script
+make install-conda-requirements
+```
+
+If you encounter any error with 'charset_normalizer' for example:
+```shell script
+AttributeError: partially initialized module 'charset_normalizer' has no attribute 'md__mypyc' (most likely due to a circular import)
+```
```

Comment:
For me I encountered this error only after running tests, so I think it worth adding a step prior or together with charset_normalizer to run tests just to see that things passes and if not how to fix them, as it will be counter intuitive to get back to the Contributing docs if failing on this